recipe/meta.yaml

```diff
@@ -9,23 +9,35 @@ source:
   git_rev: v{{ version }}
 
 build:
-  number: 3
+  number: 4
   noarch: python
   string: py_pb{{ protobuf | replace(".*", "")}}_{{ PKG_BUILDNUM }}
+  ignore_run_exports:
+    - libgcc-ng
+    - libstdcxx-ng
   
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - bazel {{ bazel }}
+    # Use pins to control cos6/cos7 match
+    - libgcc-ng  {{ libgcc }}
+    - libstdcxx-ng  {{ libstdcxx }}
   host:
     - git >=2.2
     - python  # noarch package; don't tie to specific python version
+    # Use pins to control cos6/cos7 match
+    - libgcc-ng  {{ libgcc }}
+    - libstdcxx-ng  {{ libstdcxx }}
   run:
     - python  # noarch package; don't tie to specific python version
     - numpy {{ numpy }}
     - protobuf {{ protobuf }}
     - tensorflow-base {{ tensorflow }}
+    # Use pins to control cos6/cos7 match
```

Comment:
So this recipe is a bit strange. There are compilers here, but its also noarch, we certainly don't need to add the pins to `run:` here and perhaps we don't need this PR at all.